<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ListView</class>
 <widget class="QWidget" name="ListView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1518</width>
    <height>1171</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ListView</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Preferred</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>43</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="PatientTileInfo" name="patientInfoTile" native="true">
       <property name="minimumSize">
        <size>
         <width>700</width>
         <height>150</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>100</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Номер на амб. лист:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="LeadingZeroSpinBox" name="ambNumSpin">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>999999</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="HoverButton" name="nrnButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>140</width>
           <height>26</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Дата:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="CustomDateTimeEdit" name="dateTimeEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>0</height>
          </size>
         </property>
         <property name="displayFormat">
          <string>dd.MM.yyyy HH:mm</string>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_7">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="spec_label">
         <property name="text">
          <string>Спецификация:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="specCombo">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <item>
          <property name="text">
           <string>Изцяло или частично заплащана от НЗОК</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Изцяло заплащана от НЗОК</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Обща анестезия при психични заболявания</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="unfavCheck">
         <property name="text">
          <string>Неблагоприятни условия</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="HoverButton" name="pentionTaxButton">
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>26</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Само за потребителска такса - пенсионери!</string>
         </property>
         <property name="text">
          <string>Опис на касов бон</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="IconButton" name="nssiButton">
         <property name="minimumSize">
          <size>
           <width>35</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>35</width>
           <height>35</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Проверка за право на пенсия</string>
         </property>
         <property name="text">
          <string>Пенсия</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_8">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="topMargin">
        <number>15</number>
       </property>
       <item>
        <spacer name="horizontalSpacer_11">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QFrame" name="frame">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>1041</width>
           <height>501</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>1041</width>
           <height>501</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="priceLabel">
          <property name="geometry">
           <rect>
            <x>120</x>
            <y>730</y>
            <width>521</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="ControlPanel" name="controlPanel" native="true">
          <property name="geometry">
           <rect>
            <x>760</x>
            <y>100</y>
            <width>271</width>
            <height>391</height>
           </rect>
          </property>
         </widget>
         <widget class="GraphicsView" name="teethView">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>751</width>
            <height>481</height>
           </rect>
          </property>
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
         </widget>
         <widget class="SurfacePanel" name="surfacePanel" native="true">
          <property name="geometry">
           <rect>
            <x>760</x>
            <y>10</y>
            <width>271</width>
            <height>141</height>
           </rect>
          </property>
         </widget>
         <zorder>controlPanel</zorder>
         <zorder>teethView</zorder>
         <zorder>surfacePanel</zorder>
         <zorder>priceLabel</zorder>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_12">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <property name="topMargin">
        <number>10</number>
       </property>
       <item>
        <spacer name="horizontalSpacer_9">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>90</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="procedureLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>МАНИПУЛАЦИИ:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>6</number>
           </property>
           <property name="bottomMargin">
            <number>6</number>
           </property>
           <item>
            <widget class="IconButton" name="addProcedure">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>35</width>
               <height>35</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>35</width>
               <height>35</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Добави манипулация</string>
             </property>
             <property name="text">
              <string>Добави</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="IconButton" name="editProcedure">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>35</width>
               <height>35</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>35</width>
               <height>35</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Редактирай манипулация</string>
             </property>
             <property name="text">
              <string>Редактирай</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="IconButton" name="deleteProcedure">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>35</width>
               <height>35</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>35</width>
               <height>35</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Изтрий манипулация</string>
             </property>
             <property name="text">
              <string>Изтрий</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="IconButton" name="historyButton">
             <property name="minimumSize">
              <size>
               <width>35</width>
               <height>35</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>35</width>
               <height>35</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Онлайн пациентско досие</string>
             </property>
             <property name="text">
              <string>Онлайн пациентско досие</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Expanding</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>100</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="IconButton" name="prescrButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>35</width>
               <height>35</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>35</width>
               <height>35</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Изпиши рецепта</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="IconButton" name="invoiceButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>35</width>
               <height>35</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>35</width>
               <height>35</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Издай фактура</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="IconButton" name="perioButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>35</width>
               <height>35</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>35</width>
               <height>35</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Нов пародонтален статус</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="TableView" name="procedureTable">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>850</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_10">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>90</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <property name="topMargin">
        <number>20</number>
       </property>
       <item>
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="UserButton" name="addRefButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>Добави направление</string>
         </property>
         <property name="popupMode">
          <enum>QToolButton::InstantPopup</enum>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextOnly</enum>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_15">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>650</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <spacer name="horizontalSpacer_13">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="RefLayout">
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <spacer name="horizontalSpacer_14">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="referralLayout"/>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Preferred</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>43</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PatientTileInfo</class>
   <extends>QWidget</extends>
   <header>View/SubWidgets/PatientTileInfo.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SurfacePanel</class>
   <extends>QWidget</extends>
   <header>View/SubWidgets/SurfacePanel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TableView</class>
   <extends>QTableView</extends>
   <header>View/uiComponents/TableView.h</header>
  </customwidget>
  <customwidget>
   <class>GraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>View/Graphics/GraphicsView.h</header>
  </customwidget>
  <customwidget>
   <class>IconButton</class>
   <extends>QPushButton</extends>
   <header>View/uiComponents/iconbutton.h</header>
  </customwidget>
  <customwidget>
   <class>LeadingZeroSpinBox</class>
   <extends>QSpinBox</extends>
   <header>View/uiComponents/leadingzerospinbox.h</header>
  </customwidget>
  <customwidget>
   <class>ControlPanel</class>
   <extends>QWidget</extends>
   <header>View/SubWidgets/ControlPanel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>UserButton</class>
   <extends>QToolButton</extends>
   <header location="global">View/uiComponents/userbutton.h</header>
  </customwidget>
  <customwidget>
   <class>HoverButton</class>
   <extends>QPushButton</extends>
   <header>View/uiComponents/hoverbutton.h</header>
  </customwidget>
  <customwidget>
   <class>CustomDateTimeEdit</class>
   <extends>QDateTimeEdit</extends>
   <header>View/uiComponents/customdatetimeedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
